pages/reader/epub.vue

```diff
@@ -595,6 +598,17 @@ async function loadEPub() {
   setTTSSegments(ttsSegments)
   setChapterTitles(chapterTitles)
 }
+function findNextCFIAfterTOC(navItems: NavItem[]): string | undefined {
+  const firstChapter = navItems[0]
+  if (!firstChapter) return undefined
+
+  const tocKeywords = ['目錄', '目录', 'contents', 'table of contents', 'toc', 'index']
+  const isTOC = tocKeywords.some(keyword =>
+    firstChapter.label.toLowerCase().includes(keyword),
+  )
+
+  return (isTOC && navItems[1]?.href) || firstChapter.href
```

Comment:
```suggestion
  if  (isTOC && navItems[1]?.href)  {
    return navItems[1].href
  }
  return firstChapter.href
```
Seems a explicit `if` would be more suitable since `isTOC` is really a logical condition, not a var check

Reply:
[🎨 Update return value in findNextCFIAfterTOC](https://github.com/likecoin/3ook-com/pull/449/commits/239cb947cb5623227baae31cd6c6686b37e60f93)